core/templates/pages/library-page/library-page.component.ts

```diff
@@ -329,7 +361,8 @@ export class LibraryPageComponent {
           setTimeout(() => {
             let actualWidth = $('oppia-exploration-summary-tile').width();
             if (actualWidth &&
-              actualWidth !== AppConstants.LIBRARY_TILE_WIDTH_PX) {
+              actualWidth !== AppConstants.LIBRARY_TILE_WIDTH_PX &&
+              !this.libraryWindowIsNarrow) {
```

Comment:
An error is supposed to be thrown if the width of the tile (`actualWidth`) does not match the `LIBRARY_TILE_WIDTH_PX` constant. For the mobile UI however, I have used cards of different width, so this error gets thrown.

To prevent this, I have added an extra condition, so that when the tile width is not matching, the error is only thrown if the page is not in mobile view.

I think this is an Ok workaround, but I'd like to know if maybe there's a better solution for this.

Reply:
I am supposing you are keeping the width of every tile for mobile view constant. So, you can add a new constant to constant.ts, something like `LIBRARY_TILE_WIDTH_PX_FOR_MOBILE_VIEW` and use a or operator in this if statement.

`if (actualWidth && (actualWidth !== AppConstants.LIBRARY_TILE_WIDTH_PX || actualWidth != AppConstants.LIBRARY_TILE_WIDTH_PX_FOR_MOBILE_VIEW))`

Reply:
I've added a constant for the mobile tiles, Done!